manual/manual.adoc

```diff
@@ -570,6 +570,14 @@ suggestions:
   easier for you to target. It also encourages monster in-fighting; if done
   effectively, they’ll spend their energy killing each other and you’ll save
   on ammunition.
+  
+* If you encounter a horde of flesh worms or stealth worms, The angle grinder is a
+  great weapon to use both to conserve ammo and avoid taking damage.
+  Worms can't attack while being sawed, and if you back 
+  into any corner that is roughly as wide as or narrower than a right angle,
+  they can only come at you 
+  one at a time. The angle grinder also works well on Trilobites: they can't attack
```

Comment:
use lowercase for "trilobite", it isn't a proper noun.

Reply:
Resolved in 633c7c8